Sources/protoc-gen-swift/Version.swift

```diff
@@ -17,6 +17,16 @@ struct Version {
     static let minor = 9
     static let revision = 28
     static let versionString = "\(major).\(minor).\(revision)"
+
+    // The "compatibility version" of the runtime library. This should be less
+    // than or equal to (major * 10000 + minor * 100 + revision). We guarantee
```

Comment:
Should we not even hit at a tie between the two numbers?  After all, we don't want to have to change this, it is something we'll have to do if we make breaking changes to the generated|runtime api interfaces.